### PR TITLE
installed foundation and set up styling structure

### PR DESCRIPTION
**Pull request by jspencersharpe**

### CODE REVIEW

**Comment by jspencersharpe** [comment]
this should do it

File: package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "express": "^4.13.3",
     "jade": "^1.11.0",
     "mongodb": "^2.0.41",
-    "morgan": "^1.6.1"
+    "morgan": "^1.6.1",
+    "node-sass-middleware": "^0.9.0"
   },
   "devDependencies": {
```
Comment (sscotth) [inline package.json]: bower should be a devDependency

File: www/stylesheets/main.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: blue
```
Comment (sscotth) [inline www/stylesheets/main.scss]: This style isn't showing up for me

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "foundation": "~5.5.2"
```
Comment (sscotth) [inline bower.json]: Are we initializing foundation anywhere?

File: app/index.js [hunk 4]
```diff
@@ -15,6 +16,8 @@ app.set('view engine', 'jade');
 
 app.locals.title = 'MiniTwit';
 
+app.use(sassCSS('www/stylesheets'));
```
Comment (sscotth) [inline app/index.js]: The examples on the node-sass-middleware page include more than just a string here

```
var sassMiddleware = require('node-sass-middleware');
...
app.use(sassMiddleware({
    /* Options */
    src: __dirname,
    dest: path.join(__dirname, 'public'),
    debug: true,
    outputStyle: 'compressed',
    prefix:  '/prefix'  // Where prefix is at <link rel="stylesheets" href="prefix/style.css"/>
}));
```

File: app/index.js [hunk 5]
```diff
@@ -15,6 +17,16 @@ app.set('view engine', 'jade');
...
 app.locals.title = 'MiniTwit';
 
+app.use(sassCSS({
+  src: 'www/stylesheets',
+  dest: 'www/stylesheets',
+  debug: true,
+  outputStyle: 'compressed',
+  prefix: '/style'
+}));
+
```
Comment (sscotth) [inline app/index.js:26]: remove this space please

**Comment by jspencersharpe** [comment]
passing now. can i haz merge?

**Comment by jspencersharpe** [comment]
your turn

File: app/templates/partials/stylesheets.jade [hunk 6]
```diff
@@ -0,0 +1,2 @@
+link(rel='stylesheet', href='style/main.css')
```
Comment (sscotth) [inline app/templates/partials/stylesheets.jade]: add a leading slash

**Comment by sscotth** [comment]
Need to delete bower.json, but I'll handle that.

**Comment by sscotth** [comment]
Need to delete bower.json. Also, it doesn't work, when I browse to /contact I get a 404 `GET /styles/main.css 404`

**Comment by sscotth** [comment]
Merged 08a6e0af91586fd5743b5d7e483d948d191dad6c

**Pull request closed without merging**